api/src/main/java/org/openmrs/PersonName.java

```diff
@@ -40,7 +38,6 @@
  * A Person can have zero to n PersonName(s).
  */
 @Indexed
-public class PersonName extends BaseChangeableOpenmrsData implements java.io.Serializable, Cloneable, Comparable<PersonName> {
```

Comment:
Did you just remove the class declaration by mistake?
Are you sure this class compiles?